tests/unit/test_controlTypes.py

```diff
@@ -15,13 +15,13 @@ class TestLabels(unittest.TestCase):
 
 	def test_roleLabels(self):
 		"""Test to check whether every role has its own label in controlTypes.roleLabels"""
-		for name, const in controlTypes.__dict__.iteritems():
+		for name, const in controlTypes.__dict__.items():

```

Comment:
I think I prefer this to be changed as follows while at it

```suggestion
		for name, const in vars(controlTypes).items():
```